<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetOut</class>
 <widget class="QDialog" name="GetOut">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>581</width>
     <height>371</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>备注：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="moreOut">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="numberPutOut">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>数量：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="namePutOut">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>名称：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Out">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>出库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cencleOut">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>280</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="deleteInformation">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>出库并删除商品信息</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
